webapp/components/Group/GroupForm.vue

```diff
@@ -156,11 +208,12 @@ export default {
     },
   },
   data() {
-    const { name, slug, groupType, about, description, actionRadius, locationName, categories } =
+    const { name, slug, groupType, goal, description, actionRadius, locationName, categories } =
       this.group
```

Comment:
`goal` is called `about` in the database. So this is not working …

Reply:
changed